page/about/litestar-releases.rst

```diff
@@ -1,92 +1,82 @@
-----------------
+.. note:: This schedule is flexible and may change based on community feedback and development progress.
 
-Litestar follows a non-strict release schedule, targeting 4 weeks between minor versions
-and about 1 year between major versions. For major versions, this is to be interpreted
-as a lower bound.
+Deprecation Policy
+------------------
 
+When a feature is going to be removed, a deprecation warning will be added in a **minor** release.
+The feature will continue to work for all releases in that major series, and will be removed in the next major release.
 
-About major version
---------------------
+For example, if a deprecation warning is added in ``1.1``, the feature will work throughout all ``1.x`` releases,
+and be removed in ``2.0``.
 
-Starting with version 2.0, Litestar's major releases are
-*generally backwards compatible* and usually won't include major breaking changes. They
-can be seen as maintenance releases that offer the opportunity to make some backwards
-incompatible changes.
+Supported Versions
+------------------
 
-Due to the frequency of major releases, Litestar evolves gradually over time and there
-won't be sweeping changes that require a complete rewrite of an application or major
-migration efforts when upgrading to a new major version.
+At any time, the Litestar team will actively support:
 
+- The current major release series
+- The previous major release series
+- Any other designated LTS releases
 
-Supported versions
-------------------
+For example, if the current release is ``2.0``, we will actively support ``2.x`` and ``1.x``.
+When ``3.0`` is released, we will drop support for ``1.x``.
 
-Current version
-    The current version is the last release of the most recent major version. This
-    version is under active development and will receive bugfixes as well as feature
-    updates in minor releases (see `Version numbering`_)
+Bugfixes will be applied to the current major release, and selectively backported to older
+supported versions based on severity and feasibility.
 
-Maintenance versions
-    When a new major version is released, the last *minor* version before it enters
-    maintenance mode. It will receive bugfixes and other critical patches during the
-    next two release cycles
+Release Process
+---------------
 
+Each major release cycle consists of a few phases:
 
-In practice this means that, at any given time, there may be up to 3 currently supported
-releases: The current version and the two major versions preceding it.
+#. **Planning**: Define roadmap, spec out major features. Work should begin on implementation.
+#. **Development**: Active development on planned features. Ends with an alpha release and branch of ``A.B.x``
+   branch from `main`.
+#. **Bugfixes**: Only bugfixes, no new features. Progressively release beta, release candidates.
+   Feature freeze at RC. Become more selective with backports to avoid regressions.
 
+Business / Enterprise Support
```

Comment:
This needs consideration on what this means from the @litestar-org/maintainers.

To me, when I was drafting it, it means that if `Business A` has a Litestar v2 app, and we move from `v3` -> `v4`, and they have an issue in their now-non-supported `v2` app, that we could potentially help with bug fixes or similar.

This isn't to necessarily imply "Support for monetary things", just that "We know enterprise is slow as shit so if you are stuck on v2 then we can accommodate an extended LTS for ~6-12 extra months"